app/modules/booking/bookTickets/bookTicket.controller.js

```diff
@@ -6,13 +6,17 @@ const bookTicket = async(req, res) => {
     try {
         let validation = validationResult(req);
         if (!validation.isEmpty()) {
+            console.log("%%%", error);
```

Comment:
Please ensure the removal of unwanted console.log throughout the application placed for testing purpose